### PR TITLE
add --output-deb-name option to control deb output file name

### PR DESCRIPTION
**Pull request by kmohrf**
this allows node-deb to control the name of the deb file created by dpkg-deb

### CODE REVIEW

File: node-deb [hunk 1]
```diff
@@ -352,6 +358,15 @@ if [ -z "$package_name" ]; then
...
 log_debug "The package name has been set to: $package_name"
 
+# Set the output deb name
+if [ -z "$output_deb_name" ]; then
+  output_deb_name=$(jq -r '.node_deb.output_deb_name' package.json)
+  if [[ "$output_deb_name" == 'null' ]]; then
+    output_deb_name=""
+  fi
+fi
+log_debug "The debian  name has been set to: ${output_deb_name:-dbkg-deb default}"
```
Comment (heartsucker) [inline node-deb]: Pedantic: `debian  name` has two spaces.

File: node-deb [hunk 2]
```diff
@@ -352,6 +358,15 @@ if [ -z "$package_name" ]; then
 fi
 log_debug "The package name has been set to: $package_name"
 
+# Set the output deb name
+if [ -z "$output_deb_name" ]; then
+  output_deb_name=$(jq -r '.node_deb.output_deb_name' package.json)
```
Comment (heartsucker) [inline node-deb]: Can you add double quotes around this.

File: node-deb (hunk 1 above)
Comment (kmohrf) [inline node-deb]: pedantic but true ;)

**Comment by kmohrf** [comment]
fixed

**Pull request merged**